views/content/content.html

```diff
@@ -15,25 +15,24 @@ <h2 class="base--h2">Revisions</h2>
 
 {% if content %}
 
-  <table class="table--style">
+  <table class="base--table">
     <thead class="base--thead">
       <tr class="base--tr">
         <th class="base--th">revision</th>
         <th class="base--th">date</th>
         <th class="base--th">{{config.base}}</th>
-        <th class="base--th">{{config.actions.edit}}</th>
-        <th class="base--th">{{config.actions.approve}}</th>
+        <th class="base--th" colspan="2">Actions</th>
       </tr>
     </thead>
     <tbody class="base--tbody">
     {% for c in content %}
       {% if c.value %}
         <tr class="base--tr">
-          <td class="base--td">{{c.revision}}</td>
+          <td data-th="Revision" class="base--td">{{c.revision}}</td>
           <td data-th="date" class="base--td">{{c.created}}</td>
```

Comment:
Can you please create stories for these? It should be in punchcard issues. Link stories in the issue cue below for tracking purposes.